src/toil/test/provisioners/gceProvisionerTest.py

```diff
@@ -218,11 +218,11 @@ def testAutoScale(self):
     @needs_google
     def testSpotAutoScale(self):
         self.instanceTypes = ["n1-standard-2:%f" % self.spotBid]
-        self.numWorkers = ['2']
+        self.numWorkers = ['3']
```

Comment:
why?

Reply:
In case one doesn't start and fails the test due to a transient error.

Reply:
I feel like this should be a comment in the code.